File: Basic-Video-Chat-ConnectionService_FCM/app/src/main/java/com/tokbox/sample/basicvideochat_connectionservice/AudioDeviceSelectionListener.java
package com.tokbox.sample.basicvideochat_connectionservice;

public interface AudioDeviceSelectionListener {
    /**
     * Is called when the user selects an audio device.
     * @param device The selected audio device.
     */
    void onAudioDeviceSelected(AudioDeviceSelector.AudioDevice device);
}

File: Basic-Video-Chat-ConnectionService-Kotlin/app/src/main/java/com/tokbox/sample/basicvideochatconnectionservice/deviceselector/AudioDeviceSelectionListener.kt
package com.tokbox.sample.basicvideochatconnectionservice.deviceselector

interface AudioDeviceSelectionListener {
    /**
     * Is called when the user selects an audio device.
     * @param device The selected audio device.
     */
    fun onAudioDeviceSelected(device: AudioDeviceSelector.AudioDevice)
}